src/Akeneo/Pim/Enrichment/Component/Product/Job/ComputeFamilyVariantStructureChangesTasklet.php

```diff
@@ -25,43 +26,16 @@
  */
 class ComputeFamilyVariantStructureChangesTasklet implements TaskletInterface
 {
-    /** @var StepExecution */
-    private $stepExecution;
+    private StepExecution $stepExecution;
```

Comment:
It's not really the case, here it's initialized by setStepExecution function
```suggestion
    private ?StepExecution $stepExecution = null;
```